homeassistant/components/plugwise/climate.py

```diff
@@ -87,11 +86,9 @@ def target_temperature(self) -> float | None:
         return self.device["sensors"].get("setpoint")
 
     @property
-    def hvac_mode(self) -> str:
+    def hvac_mode(self) -> str | None:
```

Comment:
None is not a valid return type for hvac mode, please revert this change

Reply:
Yeah, OK I get it now :)

Reply:
Reverted.